### PR TITLE
Add basic element interaction methods

### PR DESCRIPTION
**Pull request by dmytroap**
## Description
- added basic element interaction method `getValue`, updated `click` method
- added element state methods to get attribute and property values
- in the `typedoc.json` added `puppeteer` entry point in order to have links in the API documentation to the appropriate interfaces

### CODE REVIEW

File: src/element/actions/click.ts [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+): Promise<void> {
+  const element = await getElement(
+    context,
+    selectorOrElement,
+    searchElementOptions,
+  );
+
+  if (await isDisabled(context, element)) {
+    throw new Error(
+      'Could not perform a click on an element, the button is disabled.',
```
Comment (yurynix) [inline src/element/actions/click.ts:26]: Why you say `the button` ? It might be a link 🙂

Comment (yurynix) [reply src/element/actions/click.ts:26]: I would also put the selector maybe as part of the error so it'll be easier to debug

Comment (yurynix) [reply src/element/actions/click.ts:26]: Like you do here:
https://github.com/wix-incubator/bubanai/pull/6/files#diff-e13e6a00401250cf36dd912638bc9a62c6620a5b60faef731396238a0057940eR22

**Review by yurynix — APPROVED** [review]
Thank you for working on this! 👍

File: src/element/actions/click.ts (hunk 1 above)
Comment (dmytroap) [reply src/element/actions/click.ts]: Good point with the error message.
As for the selector I will try to figure out how to get it from the `ElementHandler` in order to print in the message.

**Pull request merged**